revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
```

Comment:
You shouldn't add styles like this to a foundation class, because they'll get added to ALL rows on the page (or maybe that's what you want)?